### DISCUSSION TITLE
post tags to hashtags

### DISCUSSION THREAD

**Discussion by azizhp** [Automattic/wordpress-activitypub#284] [Ideas]
Would it be feasible to have an option that takes the wordpress tags of a post, and then appends them as hashtags to the end of the post? This will allow us to essentially get hashtags onto the posts so they are discoverable on Mastodon. Im doing it manually for now but would be ideal to leverage the existing taxonomy. Same for categories, come to think of it.

**Comment by pfefferle** [comment]
This is already possible through a template tag. You will find a documentation in the help section of the ActivityPub settings page (the small help tab in the top right corner).
